core/src/main/java/io/kestra/core/storages/StorageInterface.java

```diff
@@ -141,6 +169,13 @@ default String cachePrefix(String namespace, String flowId, String taskId, @Null
         return String.join("/", paths);
     }
 
+    default String filePrefix(String namespace) {
```

Comment:
```suggestion
    default String namespaceFilePrefix(String namespace) {
```

Let's be more accurate